changelog/3489.added.md

```diff
@@ -0,0 +1 @@
+Add dropdown field for Default Address Type in IP Address Pool form to use nodes that inherit from BuildinIPAddress
```

Comment:
I think it's too technical for the user. It's better to talk about the goal instead of the implementation